src/pages/impact/index.tsx

```diff
@@ -0,0 +1,187 @@
+        </>
+      }
+      barStatCards={[
+        {
+          bars: [
+            { label: 2019, statistic: 17 },
+            { label: 2020, statistic: 18 },
+            { label: 2021, statistic: 21 },
+          ],
+          title: "Shelters wired",
+          subtitle: "Cumulative total",
+          orientation: "vertical",
+        },
+        {
+          bars: [
+            { label: 2019, statistic: 1265 },
+            { label: 2020, statistic: 1605 },
+            { label: 2021, statistic: 1848 },
+          ],
+          title: "People serviced nightly",
+          orientation: "horizontal",
+        },
+      ]}
+      theme="gray"
+      titleColor="blue"
+    />
+    <StatsBlock
+      title="Volunteers"
+      subtitle={
+        <>
+          As an all-volunteer organization ShelterTech relies on volunteers to
+          support digital equity.{" "}
+          <Link
+            style={{
+              color: "var(--color-sheltertech-blue)",
+              textDecoration: "underline",
+            }}
+            to="/volunteer"
+          >
+            Learn more about volunteering.
+          </Link>
+        </>
+      }
+      statCards={[
+        {
+          number: "40",
+          statement: "Active volunteers",
+        },
+        {
+          number: "15",
+          statement: "Corporate volunteer organizations",
+        },
+      ]}
+      theme="white"
+      titleColor="blue"
+    />
+    <StatsBlock
+      title="Funding"
+      subtitle={
+        <>
+          As a non-profit organization that does not focus on fundraising events
+          or similar methods to fund our work, we depend on grants from the City
+          of San Francisco and other organizations, as well as donations from
+          corporations and individuals.{" "}
+          <Link
+            style={{
+              color: "var(--color-sheltertech-blue)",
+              textDecoration: "underline",
+            }}
+            to="/donate"
+          >
+            Help us reach our next funding goal.
+          </Link>
+        </>
+      }
+      statCards={[
+        {
+          number: "$230k",
+          statement: "Total funding in 2021 (grants and donations)",
+        },
+        {
+          number: "$150k",
+          statement: "In-kind contributions in 2021",
+        },
+      ]}
+      theme="gray"
+      titleColor="blue"
+    />
+    <StatsBlock
+      title="Hosted Partners"
+      subtitle="ShelterTech is proud to provide the data on the SF Service Guide platform for other organizations to use on their own websites. We have worked closely with SF Families and Link-SF to provide a custom experience catered to their users’ needs, leveraging our data curation pipeline to keep that information accurate and up to date."
+      statCards={[
+        {
+          number: <img src={linkSFLogo} alt="Link-SF Logo." />,
+        },
+        {
+          number: <img src={sfFamiliesLogo} alt="SF Families Logo." />,
+        },
```

Comment:
Nit: Could you swap the two photos? The paragraph above mentions them in the opposite order, and it's a little bit distracting to see the logos in a different order.